Backpack-SwiftUI/FlightLeg/Classes/BPKFlightLeg.swift

```diff
@@ -0,0 +1,177 @@
+import SwiftUI
+import Backpack_Common
+
+struct BPKFlightLeg: View {
+    private let airlineLogoSize: CGFloat = 24.0
+
+    private let departureArrivalTime: String
+    private let nextDayArrival: String?
+    private let flightDescription: AttributedString
+    private let stopsInfo: String
+    private let highlightStopsInfo: Bool
+    private let duration: String
+    private let operatedBy: String?
+    private let warning: String?
+    @State private var carrierLogo: Image?
+
+    // String-based init
+    public init(
+        departureArrivalTime: String,
+        nextDayArrival: String? = "",
+        flightDescription: String,
+        stopsInfo: String,
+        highlightStopsInfo: Bool,
+        duration: String,
```

Comment:
should this not be a number? if not, what's the expected formate of this string?